src/include/connection.h

```diff
@@ -266,6 +266,9 @@ struct __wt_connection_impl {
 
     WT_TXN_GLOBAL txn_global; /* Global transaction state */
 
+    /* Recovery checkpoint base write gen */
```

Comment:
Can you place this new field where the `base_write_gen` is present? Also, I am not sure the `recovery` name suits here. How about `last_base_write_gen` with a comment of `last restart connection's base write generation` or something to indicate earlier restart?

Reply:
Updated the variable name to `last_ckpt_base_write_gen` as it was part of the last checkpoint.